CHANGELOG.md

```diff
@@ -60,6 +60,7 @@
 - Minor: Added `/streamlink` command. Usage: `/streamlink <channel>`. You can also use the command without arguments in any twitch channel to open it in streamlink. (#2443, #2495)
 - Minor: Humanized all numbers visible to end-users. (#2488)
 - Minor: Added a context menu to avatar in usercard. It opens on right-clicking the avatar in usercard. (#2517)
+- Minor: /setgame command will use loose matching, so the exact game name is not needed (#2609)
```

Comment:
Instead of adding another entry, just add the pr number to the existing changelog entry: `- Major: Add `/settitle` and `/setgame` commands, originally made for Mm2PL/Dankerino. (#2534, #2609)`